script/single-deployment/CowAmmModule.s.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.8.0 <0.9.0;
+
+import {console} from "forge-std/Script.sol";
+
+import {CowAmmModule} from "src/CowAmmModule.sol";
+
+import {Utils} from "script/libraries/Utils.sol";
+
+contract DeployCowAmmModule is Utils {
+    address internal handler;
+
+    constructor() {
+        solutionSettler = addressEnvOrDefault("SETTLEMENT_CONTRACT", DEFAULT_SETTLEMENT_CONTRACT);
+        extensibleFallbackHandler =
+            addressEnvOrDefault("EXTENSIBLE_FALLBACK_HANDLER_CONTRACT", DEFAULT_EXTENSIBLE_FALLBACK_HANDLER_CONTRACT);
+        composableCow = addressEnvOrDefault("COMPOSABLE_COW_CONTRACT", DEFAULT_COMPOSABLE_COW_CONTRACT);
+        handler = addressEnvOrDefault("CONSTANT_PRODUCT_CONTRACT", address(0));
```

Comment:
Nit: I'd move this down when it's used: it blends in with the other commands but it's logically different. I didn't notice the difference when first reading until later in my review.